sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnVectorUtils.java

```diff
@@ -54,6 +54,8 @@ public static void populate(WritableColumnVector col, InternalRow row, int field
     } else {
       if (t == DataTypes.BooleanType) {
         col.putBooleans(0, capacity, row.getBoolean(fieldIdx));
+      } else if (t == DataTypes.BinaryType) {
+        col.putByteArray(0, row.getBinary(fieldIdx));
```

Comment:
Hm? If there is no case for BinaryType, seems there is also no final `else` block, so it just leaves as unpopulate previously?

Reply:
Should we add `else` block, and throw an exception?

Reply:
> Should we add `else` block, and throw an exception?

Uopdated

Reply:
> Hm? If there is no case for BinaryType, seems there is also no final `else` block, so it just leaves as unpopulate previously?

It looks like it does, so that value is empty